### PR TITLE
Add support for the Zig programming language

### LINKED ISSUE #91: Support deleting the Zig cache
Hey, thanks for this tool! 🐻 

I realized `kondo` doesn't delete the Zig cache folder which is usually `zig-cache`.

If appropriate, I would like to submit a PR for this!

### PR DESCRIPTION
**Pull request by orhun**
closes #91

### CODE REVIEW

File: kondo-lib/src/lib.rs [hunk 1]
```diff
@@ -237,11 +242,13 @@ impl Project {
             ProjectType::Elixir => PROJECT_ELIXIR_NAME,
             ProjectType::Swift => PROJECT_SWIFT_NAME,
             ProjectType::Gradle => PROJECT_GRADLE_NAME,
+            ProjectType::Zig => PROJECT_ZIG_NAME,
         }
     }
 
     /// Deletes the project's artifact directories and their contents
     pub fn clean(&self) {
+        println!("{:?}", self.artifact_dirs());
```
Comment (tbillington) [inline kondo-lib/src/lib.rs:251]: Looks like this was left in by accident :)

**Review by tbillington — CHANGES_REQUESTED** [review]
Just that accidental print statement to remove but otherwise looks great, thank you!

File: kondo-lib/src/lib.rs (hunk 1 above)
Comment (orhun) [reply kondo-lib/src/lib.rs:251]: Oops, sorry. I'm at an airport with 2hrs of sleep. Will fix ASAP!

Comment (tbillington) [reply kondo-lib/src/lib.rs:251]: No rush :)

Comment (orhun) [reply kondo-lib/src/lib.rs]: PR is updated!

**Comment by tbillington** [comment]
🚀

**Pull request merged**